lib/cell-manager.coffee

```diff
@@ -0,0 +1,51 @@
+_ = require 'lodash'
+
+module.exports = CellManager =
+    removeAllBreakPoints: ->
+        editor = atom.workspace.getActiveTextEditor()
+        decorations = editor.getLineNumberDecorations({class: 'break-point'})
+        for decoration in decorations
+            decoration.marker.destroy()
+
+    removeLatestBreakPoint: ->
+        editor = atom.workspace.getActiveTextEditor()
+        decorations = editor.getLineNumberDecorations({class: 'break-point'})
+        if decorations.length isnt 0
+            _.last(decorations).marker.destroy()
+
+    addBreakPoint: ->
+        editor = atom.workspace.getActiveTextEditor()
+        row = editor.getLastCursor().getBufferRow()
+
+        marker = editor.markBufferPosition
+            row: row
+            column: 0
+            invalidate: 'never'
+
```

Comment:
Hmm it doesn't throw an error.

I removed the brackets because of your [comment](https://github.com/nteract/hydrogen/pull/227#discussion_r57532735).
What would be the right way for this in coffeescript?